### ISSUE TITLE
Create a Website for this Repo

### ISSUE THREAD

**Issue by xdvrx1** [bsamseth/cpp-project#29]
Hi there. 

Is there a website for this repo? 

Because if you don't have, well, this repo can simply be turned into a website right away. Others will discover this project in that website.

Steps:
1. Go to Settings and look for `GitHub Pages`, scroll down. That's almost at the bottom.

2. You will see there: `Branch:none`, so you should change that to `master` because you have a `README.md` file in the master repo. This will be your page. Click `Save` first.

3. Then click `Choose a theme`, you select a predefined theme of your site.

4. Visit your site now! The URL will be `https://bsamseth.github.io/cpp-project`. 

If you were amazed by that, simply read the documentation about GitHub Pages.

**Comment by bsamseth** [comment]
Thank you for raising an issue. I'm well aware of GitHub Pages and how to enable it. However, I don't feel like this repo benefits in any way from having one because the main point here is the actual code structure, which would not be available on such a webpage anyway.

Also, the notion that people would discover the repo via the webpage is unlikely. The page really wouldn't exist until Google decides to surface it, which would only happen if sufficiently many visitors to the repo (i.e. the only place linking to the site) wander off to the website (which would be a waste of time for those users). Meanwhile, this repo has way more hits and surfaces much better in search.

In short, thank you for suggesting it, but I don't believe there is a benefit for this particular repo.
